main.go

```diff
@@ -398,9 +398,18 @@ This means that when the API changes
 	}
 
 	if configs.IpaPath != "" {
-		args = append(args, "--ipa", configs.IpaPath)
+		tmpIpaPath, err := copyToTmp(configs.IpaPath)
+		if err != nil {
+			fail("Deploy failed, error: %s", err)
```

Comment:
do not break the step if our fix failed, instead keep using the original path and print a warning message.